### PR TITLE
fix(theme): searchbar should have normal size input

### PR DESCRIPTION
**Pull request by Haroenv**
Usually you want the searchbox to have the same font size (at least) as your body text. Otherwise it'll look awkardly small without other styling.

Also fixed a typo in `appearance` and deduplicated two properties

before|after
---|---
screen shot 2017-11-01 at 07 50 15 | screen shot 2017-11-01 at 07 50 27

### CODE REVIEW

**Comment by samouss** [comment]
@Haroenv You should update your commit message, the validation fail on the last two.

**Comment by Haroenv** [comment]
I can’t @samouss, committed on my phone and can’t rebase

**Comment by samouss** [comment]
You can do it later, don't worry 😉

**Comment by bobylito** [comment]
Can you update the commit descriptions?

**Comment by Haroenv** [comment]
sure, done

File: src/css/theme/_search-box.scss [hunk 1]
```diff
@@ -1,15 +1,13 @@
 .ais-search-box {
   display: inline-block;
   position: relative;
-  max-width: 300px;
-  width: 100%;
```
Comment (bobylito) [inline src/css/theme/_search-box.scss]: How to make sure we don't break current implementations with this change?

Comment (Haroenv) [reply src/css/theme/_search-box.scss]: this is a duplicate declaration, already set in the main css

File: src/css/theme/_search-box.scss [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
   height: 46px;
   white-space: nowrap;
-  box-sizing: border-box;
```
[the remark this answers is not in the document]
Comment (Haroenv) [reply src/css/theme/_search-box.scss]: nope, it's already set for all ais-

**Pull request merged**